Allow the text in the user list hostname column to be changed

culb opened the pull request:
Allow the text in the user list hostname column to be changed

Commit:
Add a method to the plugin API

Add a method to the plugin API to directly send to a server.

Commit:
Add a method to the plugin API

Add a method to the plugin API to directly send to a server.

Commit:
Add hexchat_send_raw to the Lua plugin

Add the hexchat_send_raw method to the Lua plugin. This refers to commit
3be5c45.

Commit:
Add hexchat_send_raw to the Lua plugin

Add the hexchat_send_raw method to the Lua plugin. This refers to commit
3be5c45.

Commit:
Add plugin_pref_del to the Perl export list

The documentation lists "HexChat::plugin_pref_del( $setting )" as the
method to delete preferences but only plugin_pref_delete is exported.

Commit:
Add plugin_pref_del to the Perl export list

The documentation lists "HexChat::plugin_pref_del( $setting )" as the
method to delete preferences but only plugin_pref_delete is exported.

Commit:
Revert "Add a method to the plugin API"

This reverts commit 3be5c45d31bfc70c50b3aea54c05563247404845.

Commit:
Revert "Add a method to the plugin API"

This reverts commit 3be5c45d31bfc70c50b3aea54c05563247404845.

Commit:
Revert "Add hexchat_send_raw to the Lua plugin"

This reverts commit f7412d513a64f03b522a804ebf9a2d9db1011e92.

Commit:
Revert "Add hexchat_send_raw to the Lua plugin"

This reverts commit f7412d513a64f03b522a804ebf9a2d9db1011e92.

culb committed:
User list hostnames text color change

Allow the text in the user list hostname column to be changed

culb committed:
User list hostnames text color change

Allow the text in the user list hostname column to be changed

TingPing commented:
Rebase. Also I'm not sure if just telling people "Modify this undocumented internal list" is exactly a friendly extension point.

culb commented:
I made a mistake by forgetting to add color. It should be "Allow the text color in the user list hostname column to be changed"

culb committed:
Merge remote-tracking branch 'refs/remotes/hexchat/master'

culb committed:
Fix Perl script registration version

Perl knows "0" as false so if "0" is used as a version, it doesn't
recognize it as being defined

culb committed:
Fix Typo

Fix Typo

culb committed:
Merge pull request #1 from culb/PerlChangeVersion

Fix Perl script regisration version
